ChangeLog

```diff
@@ -1,52 +1,65 @@
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	version 1.6.2 and upcomming version 1.7
-	* Compatibility with Coq 8.7
+23/04/2018 - compatibility with Coq 8.7 and several small fixes -
+	upcoming version 1.7
+	* Compatibility with Coq 8.8 (and >= 8.6)
 	* Lost compatibility with Coq 8.4
 
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	upcomming version 1.7
-	* New Theorems:
-	   dec_factor_theorem, abstract_context,
-	   mul_bin_down, mul_bin_left
+	* Integration to Coq: ssrbool.v ssrfun.v and plugin.
+	  ssrtest also moved to Coq test suite.
+
+	* Library refactoring:
+	odd_order and real_closed: are now in separate repositories.
+
+	algC, complex, ssrnum: Library ssrnum.v now
+	provides an interface numClosedFieldType, which factors some
+	theory from both library complex.v (now in the real_closed
+	repository) and algC. In particular, Re, Im, 'i,
+	conjC, n.-root and sqrtC are now part of this generic
+	interface. In case of ambiguity, they can be casted to the type
+	algC, of complex algebraic numbers, via typing constraints. Some
+	theory was thus made more generic, and the corresponding lemmas,
+	previously defined in library algC.v (e.g. conjCK) now feature an
+	extra, non maximal implicit, parameter
+	(C : numClosedFieldType). This could break some
+	proofs. Additionally, a few ad hoc constructions in library
```

Comment:
Since complex.v had never been released, I wonder if it is worth mentioning?

Reply:
This paragraph was in the previous version of the ChangeLog (and was written by yoursefl?): I just made minor spelling/English editions. Since it mentioned  library complex, I felt we needed to explain what happened or withdraw everything. I chose the most conservative option to trigger discussion. I am very fine with removal.

Reply:
I think that things not in the (released) library should not be mentioned here

Reply:
Agreed. I'll rephrase.